pkg/ocp/ocp.go

```diff
@@ -84,48 +92,67 @@ func DumpManifests(outputDir string, manifests ocp4.Manifests) {
 	}
 }
 
-func (config *ConfigMaster) Fetch(outputDir string) {
-	localF := filepath.Join(outputDir, config.Hostname, config.Path)
-	config.ConfigFile.Content = GetFile(config.Hostname, config.Path, localF)
-	logrus.Printf("File:Loaded: %s", localF)
-}
-
-func (config *ConfigNode) Fetch(outputDir string) {
-	localF := filepath.Join(outputDir, config.Hostname, config.Path)
-	config.ConfigFile.Content = GetFile(config.Hostname, config.Path, localF)
+func Fetch(configFile *ConfigFile) {
+	localF := filepath.Join(env.Config().GetString("OutputDir"), configFile.Hostname, configFile.Path)
+	configFile.Content = GetFile(configFile.Hostname, configFile.Path, localF)
 	logrus.Printf("File:Loaded: %s", localF)
 }
 
-// GenYAML returns the list of translated CRDs
-func (config *ConfigMaster) GenYAML() ocp4.Manifests {
+// GenYAML returns the list of OAuth CRDs
+func (oauthConfig *OAuthConfig) GenYAML() ocp4.Manifests {
 	var manifests ocp4.Manifests
```

Comment:
I'm concerned about a general type for `ocp4.Manifests`. Have we considered the fact that more than one ocp4 feature may need to provide their configuration on the **same CR**, which means they'd need to edit a shared output file. Perhaps we need to share a data structure between transforms that represents the total file output so that more than one `Translator` can mutate the same individual output file.

Example: FooFeature and BarFeature in OCP4 both are configured by writing to the same kube object, `FooBar`. How do we deal with that?

Reply:
We are effectively not covering this use case yet.
But it might be needed so we need to anticipate. 
Will need to add it in following PR.